python/cugraph/cugraph/testing/__init__.py

```diff
@@ -54,9 +55,4 @@
     toy_graph,
     toy_graph_undirected,
 ]
-ALL_DATASETS_WGT = [karate, dolphins, netscience, polbooks, small_line, small_tree]
-TEST_GROUP = [dolphins, netscience]
-
-# FIXME: removed karate variant. check if unit tests are breaking
-DATASETS_UNRENUMBERED = []
-DATASETS = [dolphins, netscience, karate]
+DATASETS_TESTING = [dolphins, netscience, karate_disjoint]
```

Comment:
Hmmm, this name seems a little awkward IMO.  Since we're treating these as the default set of datasets to use for tests, how about `DEFAULT_DATASETS`?  Then to be consistent (if it's not too much trouble) maybe change the naming convention of the others to be `SMALL_DATASETS`, `WEIGHTED_DATASETS`, etc.?

Reply:
Sounds good. I created a table that includes all the datasets that are associated with each unit test, so that should be a simple fix.